python/cudf/cudf/tests/test_parquet.py

```diff
@@ -426,6 +427,10 @@ def test_parquet_writer_cpu_pyarrow(tmpdir, pdf, gdf):
     pdf_fname = tmpdir.join("pdf.parquet")
     gdf_fname = tmpdir.join("gdf.parquet")
 
+    if len(pdf) == 0:
+        pdf = pdf.reset_index(drop=True)
+        gdf = gdf.reset_index(drop=True)
```

Comment:
Why do we need to reset the index here? We should be able to handle round tripping a zero sized index properly.

Reply:
This stems from the same issue of previously accidentally creating a `RangeIndex` here on the pandas side. It passed before because we were actually dropping the index completely before writing out the data. Now that we actually have to compare the index, this fails because the pandas dataframe sports a bare `pd.Index` object and we get a `Float64Index`. As far as I can tell, a `cudf.DataFrame` can't exist with just a base `Index` object right now.